itests/src/test/java/org/jclouds/karaf/itests/FeatureInstallationTest.java

```diff
@@ -0,0 +1,414 @@
+   }
+
+   @Test
+   public void testGlesysFeature() throws Exception {
+      featuresService.installFeature("jclouds-glesys");
+   }
+
+   @Test
+   public void testGo2cloudJhb1Feature() throws Exception {
+      featuresService.installFeature("jclouds-go2cloud-jhb1");
+   }
+
+   @Test
+   public void testGogridFeature() throws Exception {
+      featuresService.installFeature("jclouds-gogrid");
+   }
+
+   @Test
+   public void testGoogleComputeEngineFeature() throws Exception {
+      featuresService.installFeature("jclouds-google-compute-engine");
+   }
+
+   @Test
+   public void testOpenhostingEast1Feature() throws Exception {
+      featuresService.installFeature("jclouds-openhosting-east1");
+   }
+
+   @Test
+   public void testProfitbricksFeature() throws Exception {
+      featuresService.installFeature("jclouds-profitbricks");
+   }
+
+   @Test
+   public void testRackaspaceCloudstorageUkFeature() throws Exception {
+      featuresService.installFeature("jclouds-rackspace-cloudblockstorage-uk");
+   }
+
+   @Test
+   public void testRackaspaceCloudstorageUsFeature() throws Exception {
+      featuresService.installFeature("jclouds-rackspace-cloudblockstorage-us");
+   }
+
+   @Test
+   public void testRackaspaceCloudloadbalancerUsFeature() throws Exception {
+      featuresService.installFeature("jclouds-rackspace-cloudloadbalancers-us");
+   }
+
+   @Test
+   public void testRackaspaceCloudloadbalancerUkFeature() throws Exception {
+      featuresService.installFeature("jclouds-rackspace-cloudloadbalancers-uk");
+   }
+
+   @Test
+   public void testRackaspaceCloudserversUsFeature() throws Exception {
+      featuresService.installFeature("jclouds-rackspace-cloudservers-us");
+   }
+
+   @Test
+   public void testRackaspaceCloudserversUkFeature() throws Exception {
+      featuresService.installFeature("jclouds-rackspace-cloudservers-uk");
+   }
+
+   @Test
+   public void testRackaspaceCloudDnsUsFeature() throws Exception {
+      featuresService.installFeature("jclouds-rackspace-clouddns-us");
+   }
+
+   @Test
+   public void testRackaspaceCloudDnsUkFeature() throws Exception {
+      featuresService.installFeature("jclouds-rackspace-clouddns-uk");
+   }
+
+   @Test
+   public void testRackaspaceCloudfilesUkFeature() throws Exception {
+      featuresService.installFeature("jclouds-rackspace-cloudfiles-uk");
+   }
+
+   @Test
+   public void testRackaspaceCloudfilesUsFeature() throws Exception {
+      featuresService.installFeature("jclouds-rackspace-cloudfiles-us");
+   }
+
+   @Test
+   public void testServerloveZ1ManFeature() throws Exception {
+      featuresService.installFeature("jclouds-serverlove-z1-man");
+   }
+
+   @Test
+   public void testSkalicloudSdgMyFeature() throws Exception {
+      featuresService.installFeature("jclouds-skalicloud-sdg-my");
+   }
+
+   @Test
+   public void testSoftlayerFeature() throws Exception {
+      featuresService.installFeature("jclouds-softlayer");
```

Comment:
It need some special configuration for the `verify` goal - don't think that's in place now? Let's go with the tests.